packages/wonder-blocks-dropdown/components/dropdown-core.js

```diff
@@ -9,12 +9,19 @@ import Color, {fade} from "@khanacademy/wonder-blocks-color";
 import Spacing from "@khanacademy/wonder-blocks-spacing";
 import {View} from "@khanacademy/wonder-blocks-core";
 
+import ActionItem from "./action-item.js";
```

Comment:
:shrug	I think you can actually make these `import typeof ...` and avoid having to import the whole module! (and then remove the typeofs below)

Reply:
Woahh I didn't know that. Will definitely use this instead! (It took me long enough to figure out how to do `typeof Blah` in the first place.